sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/AzureDataLakeStorageGen2DataFeedSource.cs

```diff
@@ -7,19 +7,21 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
-    /// A Data feed source.
+    /// Ingests data into a <see cref="DataFeed"/> for anomaly detection.
     /// </summary>
     public class AzureDataLakeStorageGen2DataFeedSource : DataFeedSource
     {
+        // TODODOCS.
```

Comment:
Is there more to do here?

Reply:
Yep. Missing `fileTemplate` description.